com.pot_app.pot.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.pot_app.pot</id>
+  
+  <name>Pot</name>
+  <summary>Cross-platform translation software</summary>
+  <developer_name>Pylogmon</developer_name>
+
+  <url type="homepage">https://pot-app.com/</url>
+  <url type="bugtracker">https://github.com/pot-app/pot-desktop/issues</url>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  
+  <description>
+    <p>A cross-platform software for text translation and recognize.</p>
+    <p xml:lang="zh_CN">一个跨平台的划词翻译和 OCR 软件</p>
+  </description>
+  
+  <launchable type="desktop-id">com.pot_app.pot.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/pot-app/pot-desktop/raw/master/asset/1.png</image>
```

Comment:
you shouldn't be linking to a file on `master` but rather to a specific version to ensure it be permanent.